internal/pkg/deploy/cloudformation/errors.go

```diff
@@ -23,6 +23,25 @@ func (err *ErrStackAlreadyExists) Unwrap() error {
 	return err.parentErr
 }
 
+// ErrStackNotFound occurs when we can't find a particular CloudFormation stack.
+type ErrStackNotFound struct {
+	stackName string
+}
+
+func (err *ErrStackNotFound) Error() string {
+	return fmt.Sprintf("failed to find a stack named %s", err.stackName)
```

Comment:
nit: can we removed "failed to" from the errors? `"stack %s  cannot be found"` for example

Reply:
yea yea. good call.